content/1.7/back-office/international/locations/add-new-zone.md

```diff
@@ -0,0 +1,19 @@
+---
```

Comment:
Could you create a "zones" folder with the appropriate _index.md file:
content/1.7/back-office/international/locations/zones/_index.md ?
And then move this file in the "zone" folder: content/1.7/back-office/international/locations/zones/add-new-zone.md ?